### PR TITLE
[FSPE-5271] Solid vs Base and some other minor fixes

### PR DESCRIPTION
**Pull request by gsanchezu**
Hey @jllr, @janibolkvadze, @mattnickles, could you please review? 

I tried to add some terms in bold, although I haven't seen you really using it across the doc. I've used #[] for it, but not sure if that will work in our case.

### CODE REVIEW

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://b9b8ed75e8926a79d49b868a44a837f0792d7079-chi-pr.enterprise.ctl.io" target="_blank"> https://b9b8ed75e8926a79d49b868a44a837f0792d7079-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1078/ for more details.

File: src/website/views/getting-started/development-workflow.pug [hunk 1]
```diff
@@ -145,9 +145,9 @@ p.-text Method decorator should be multi-line
...
+p.-text Use private variables and methods as much as possible; they are useful to detect dead code and enforce encapsulation. Note that this is a feature which TypeScript provides to help hardening your code, but using private, public or protected does not make a difference in the actual JavaScript output.
 
-p.-text Code with Method/Prop/Event/Component decorators should have jsdocs: This allows documentation generation and for a better user experience in an editor that has TypeScript intellisense. <em>Stencil auto-generated documentation will be included in the Chi documentation pages. </em>
+p.-text Code with Method/Prop/Event/Component decorators should have #[jsdocs]; this allows documentation generation and for a better user experience in an editor that has TypeScript intellisense. <em>Stencil auto-generated documentation will be included in the Chi documentation pages. </em>
```
Comment (mattnickles) [inline src/website/views/getting-started/development-workflow.pug:150]: Pug is rendering as an html tag and the text jsdocs is no longer visible on the page

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Build FAILED. See http://jenkins.elasticbox.com/job/chi_pull_request/1083/ for more details.

**Comment by jllr** [comment]
test this please

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://fbed7a2b8c9e21437dbb7dd4810125a8b576641e-chi-pr.enterprise.ctl.io" target="_blank"> https://fbed7a2b8c9e21437dbb7dd4810125a8b576641e-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1087/ for more details.

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-zi3qoz, https://ops.elasticbox.com/services/instances/i-xlxhyv

**Pull request merged**